build/circle-test.sh

```diff
@@ -125,7 +125,7 @@ if is_shard 0; then
   time ${builder} /bin/bash -c "(git ls-files --modified --deleted --others --exclude-standard | diff /dev/null -) || (git add -A && git diff -u HEAD && false)" | tee "${outdir}/generate.log"
 fi
 
-if is_shard 1; then
+if is_shard 0; then
```

Comment:
I want to maintain the existing test ordering in case there is no parallelism enabled on CircleCI.